### PR TITLE
[WIP] Rough MacOS Playbook

### PR DESCRIPTION
**Pull request by cwillhelm**
Currently running locally against an internal host with:

`ansible-playbook --user [redacted]  MacOS.yml --ask-become-pass`

In reference to: https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/212

Signed-off-by: Connor Willhelm <connor.willhelm@ibm.com>

### CODE REVIEW

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/MacOS.yml [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+    #  - ant-contrib
+      state: present
+    tags: brew_packages
+  
+  ##############
+  # JDK11 Only #
+  ##############
+  - name: Download and extract JDK11 
+    unarchive:
+      src: https://github.com/AdoptOpenJDK/openjdk10-releases/releases/download/jdk-10.0.2%2B13/OpenJDK10_x64_Mac_jdk-10.0.2.13.tar.gz
```
Comment (jdekonin) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/MacOS.yml]: Try how the unix playbook role does this.
https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/8be6072a398346ab2d97a3dc352fe3c4ee44ce62/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/adoptopenjdk11/tasks/main.yml#L50

**Comment by AdamBrousseau** [comment]
@cwillhelm can you add bootjdk11 (ref #594) since this is still WIP?

**Comment by cwillhelm** [comment]
@AdamBrousseau will do, I've been without a machine for testing until we get the Macs back from UNB, so the playbook has been on the backburner a bit. Once we get them back I'll start making more progress.

**Comment by gdams** [comment]
Can we not use roles here rather than pushing all of the mac stuff into a single area?

**Comment by sxa555** [comment]
I'm ok with an initial version being done as one file then splitting out into roles later to match the Linux platforms. More critical that we get "something" in place just now I think.

**Comment by sxa555** [comment]
I'm ok with an initial version being done as one file then splitting out into roles later to match the Linux platforms. More critical that we get "something" in place just now I think.

**Comment by sxa555** [comment]
@cwillhelm What is the purpose of the cache here? We don't have it on other platforms, and I don't see the reasoning documented anywhere.

**Comment by AdamBrousseau** [comment]
Repo cache is something we use in OpenJ9 builds to make clones faster. Adopt builds do not use this at the moment. Since we have a job that will automatically add & update the cache, I don't think it's necessary for the playbooks at this point.

**Comment by jdekonin** [comment]
Taken from discussion with @cwillhelm, `pmset -a sleep 0` might be of use in stopping the mac from going to sleep and becoming disabled in jenkins.

**Comment by gdams** [comment]
closing in favor of https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/826

**Pull request closed without merging**